src/applications/vaos/appointment-list/index.jsx

```diff
@@ -5,23 +5,37 @@ import { selectFeatureHomepageRefresh } from '../redux/selectors';
 import PageLayout from './components/AppointmentsPage/PageLayout';
 import AppointmentsPageV2 from './components/AppointmentsPage/AppointmentsPageV2';
 import AppointmentsPage from './components/AppointmentsPage/index';
+import RequestedAppointmentDetailsPage from './components/RequestedAppointmentDetailsPage';
 
 function AppointmentListSection({ featureHomepageRefresh }) {
   return (
-    <PageLayout>
-      <Switch>
+    <Switch>
+      {featureHomepageRefresh && (
         <Route
-          path="/"
-          render={() => {
-            if (featureHomepageRefresh) {
-              return <AppointmentsPageV2 />;
-            }
-
-            return <AppointmentsPage />;
-          }}
+          path="/request/:id"
```

Comment:
I think the normal RESTful pattern is for this to be `requests`, though we don't have a `/requests` page (it's `/requested`). Not a big deal either way, but it could be useful to be consistent for all the request urls.

Reply:
Changed to `/requests/:id`